src/app/index.tsx

```diff
@@ -1,29 +1,16 @@
 import React from 'react';
 import { SafeAreaView, StatusBar } from 'react-native';
 
-import BookDetail from './components/BookDetail';
-import { WHITE } from './constants/colors';
+import { WHITE } from '../constants/colors';
+
+import HomeScreen from './screens/HomeScreen';
```

Comment:
```
import { WHITE } from '@constants/colors';
import HomeScreen from '@screens/HomeScreen';
```

Reply:
why is linter not letting me put them together if i don't use the path shortcuts?

Reply:
Bad linter config, working correctly now 😄